cloudbuild.yaml

```diff
@@ -38,10 +39,21 @@ steps:
     secretEnv: [
       'DISCORD_SECRET_TOKEN'
     ]
+  - id: 'Deploy RAWG config'
+    dir: deploy-commands-function
+    name: golang:1.22
+    script: go run .
+    env:
+      - 'RAWG_SECRET_ID=$_RAWG_SECRET_ID'
+    secretEnv: [
+      'RAWG_SECRET_TOKEN'
+    ]
```

Comment:
Remove this. We run the `deploy-commands-function` program above to register our supported discord commands: https://discord.com/developers/docs/interactions/application-commands#registering-a-command
```suggestion
```